config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
 
-  get 'charges/create'
-
   resources :wikis
   resources :charges, only: [:new, :create]
 
   devise_for :users
+  resources :users, only: [:update, :show] do
+    post 'downgrade'
```

Comment:
the button_to creates a mini-form (you can use the chrome inspector to see the html that gets created) which submits a post request to a url.  so since you're using a button_to, this is the correct route.  however, you should run rake routes to determine what the correct named url is.  For example, you might see that the named url is downgrade_user and that there's a placeholder in the route (e.g. :id), in which case the method would be downgrade_user_path(user_object).  Or maybe there's no placeholder, or maybe the named url is just downgrade.